docker/swarm/docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     deploy:
       replicas: 1
       restart_policy:
-        condition: on-failure
+        condition: always
```

Comment:
Is `always` the correct value or should it be `any`?

I'm not that familiar with Docker Swarm, but it seems to follow the docker-compose file conventions. According to the [docker-compose docs](https://docs.docker.com/compose/compose-file/compose-file-v3/) these are the valid values for `restart_poilcy.condition`:

<img width="983" alt="image" src="https://user-images.githubusercontent.com/11049609/124809412-f0f72200-df14-11eb-9a24-8285b8c337a2.png">

Reply:
You are right, unfortunately. Apparently I hadn't tested the correct mode here. I'll open a new PR to fix this bug.

Reply:
My apologies for not getting to the review sooner. 😞